samza-core/src/main/java/org/apache/samza/zk/ZkJobCoordinator.java

```diff
@@ -157,8 +162,15 @@ public void stop() {
 
         debounceTimer.stopScheduler();
 
-        LOG.debug("Shutting down ZkController.");
-        zkController.stop();
+        LOG.info("Shutting down Zk Client.");
```

Comment:
Minor: Maybe move to where ZkUtils is being closed, and change to "Shutting down ZkUtils"?